src/instructlab/clickext.py

```diff
@@ -73,25 +73,28 @@ def get_command(self, ctx: click.Context, cmd_name: str) -> click.Command | None
             cmd = self.alias_eps[cmd_name].load()
             if typing.TYPE_CHECKING:
                 assert isinstance(cmd, click.Command)
-            group, primary = self.get_alias_info(cmd_name)
-            click.echo(
-                "You are using an aliased command, this will be deprecated "
-                "in a future release. Please consider using "
-                f"`ilab {group} {primary}` instead"
-            )
             # if some storage dirs do not exist
-            # AND the command is not `ilab init`
             # AND the --config flag is not customized, then we error
             if not storage_dirs_exist() and (
-                primary != "init" and ctx.params["config_file"] == DEFAULTS.CONFIG_FILE
+                ctx.params["config_file"] == DEFAULTS.CONFIG_FILE
             ):
                 click.secho(
                     "Some ilab storage directories do not exist yet. Please run `ilab config init` before continuing.",
                     fg="red",
                 )
                 raise click.exceptions.Exit(1)
             return cmd
-        return super().get_command(ctx, cmd_name)
+        command = super().get_command(ctx, cmd_name)
+        if command is None:
+            click.echo(
+                f"Command '{cmd_name}' not found. If this was an aliased command, please note that aliases have been deprecated in 0.19.0.",
+                err=True,
+            )
+            click.echo(
+                "Please refer to the updated documentation for the new command structure.",
+                err=True,
+            )
+        return command
```

Comment:
Do we have a test to validate this addition? Isn't this outside of the scope of removing aliases? Am I missing something?

Reply:
We are removing aliases
  * `convert`
  * `diff`
  * `download`
  * `evaluate`
  * `init`
  * `list`
  * `sysinfo`
  * `test`
  
 and keeping aliases
  * `train`
  * `generate`
  * `chat` 
  * `serve`
  
Since we are keeping some aliases, I suppose it doesn't make sense to say that aliases have been deprecated so I will remove `If this was an aliased command, please note that aliases have been deprecated in 0.19.0.` from the error.

Reply:
Resolved offline, we are removing this part. Thanks!